WordPress/src/main/java/org/wordpress/android/util/extensions/CompatExtensions.kt

```diff
@@ -18,3 +23,55 @@ fun OnBackPressedDispatcher.onBackPressedCompat(onBackPressedCallback: OnBackPre
     onBackPressed()
     onBackPressedCallback.isEnabled = true
 }
+
+/**
+ * TODO: Remove this when stable androidx.core 1.10 is released. Use IntentCompat instead.
+ */
+@Suppress("ForbiddenComment")
+inline fun <reified T : Parcelable> Intent.getParcelableExtraCompat(key: String): T? =
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        getParcelableExtra(key, T::class.java)
+    } else {
+        @Suppress("DEPRECATION")
+        getParcelableExtra(key) as T?
+    }
+
+inline fun <reified T : Serializable> Intent.getSerializableExtraCompat(key: String): T? =
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        getSerializableExtra(key, T::class.java)
+    } else {
+        @Suppress("DEPRECATION")
+        getSerializableExtra(key) as T?
+    }
+
+/**
+ * TODO: Remove this when stable androidx.core 1.10 is released. Use BundleCompat instead.
+ */
+@Suppress("ForbiddenComment")
+inline fun <reified T : Parcelable> Bundle.getParcelableCompat(key: String): T? =
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        getParcelable(key, T::class.java)
+    } else {
+        @Suppress("DEPRECATION")
+        getParcelable(key)
+    }
+
+/**
+ * TODO: Remove this when stable androidx.core 1.10 is released. Use IntentCompat instead.
```

Comment:
Minor (🔍): I believe `IntentCompat` is a typo within this KDoc, consider replacing with `BundleCompat`.

Reply:
Nice catch! Done: d16d34d395b88d39a1ba90b7ebc7762afd0efca3

Reply:
Nice update, @irfano rocks! 🙇 ❤️